### ISSUE TITLE
Dialog after editing leaves the editor blank

### ISSUE THREAD

**Issue by ericlangedijk** [JAM-Software/Virtual-TreeView#895]
When editiing and using the OnNewText event which could raise an exception, the editor is blank during the exception message. We do not see the offending wrong value which the user has typed.
Additionally the state of the editor is unpredictable after an exception: the editor is visible but not responsive anymore.

**Comment by joachimmarder** [comment]
Please respect our guidelines for reporting bugs. When reporting a bug please include a sample project that allows us to quickly reproduce the bug. This can also be one of the demo projects that come with Virtual Treeview, modified to show the bug. If only small changes are required, a description is sufficient how a demo projects needs to be changed in order to replicate the bug. If you already have a solution, please supply a patch file.

> When editiing and using the OnNewText event which could raise an exception, the editor is blank during the exception message.

A handled or an unhandled exception?

**Comment by ericlangedijk** [comment]
Sorry I will make a project to show it.
The same effect can be seen in the projectmanager of the Delphi IDE, which is - I believe - a VirtualTreeView.
Rename a unit in the tree to an invalid name and you can see the blank editor during the exception message.

**Comment by joachimmarder** [comment]
What I see in the Delphi IDE is an unhandled exception. I don't think we should optimize for that case.

**Comment by ericlangedijk** [comment]
It does not matter if there is an exception. Using the OnNewText eventhandler will leave the editor blank too when we use a ShowMessage inside this OnNewText event. Maybe because FStopping = True (?)
If needed I can make a reproduce-project for this.

**Comment by joachimmarder** [comment]
> Using the OnNewText eventhandler will leave the editor blank too when we use a ShowMessage inside this OnNewText event

Thanks, that's an easy repro.

What I found out so far: If a dialog window is shown, then an `EN_UPDATE` message is processed by the message queue and causes a call to `TVTEdit.AutoAdjustSize()`. Here the `Text` property contains a trailing `CRLF`. I guess it is this line feed that causes the text to disappear. But I did not find a solution so far.
